src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/Microsoft.NETCore.Runtime.CoreCLR.pkgproj

```diff
@@ -17,6 +17,9 @@
     <Dependency Include="Microsoft.NETCore.Windows.ApiSets">
       <Version>1.0.1-rc3-23915</Version>
     </Dependency>
+    <Dependency Include="Microsoft.NETCore.Jit">
```

Comment:
Agreed.  @jaredpar and I are working on plan for inter-repo orchestration which we'll share more broadly early next week.